tests/k8s_testcases/expected_output/orig_online_boutique_synthesis_res_connectivity_map.txt

```diff
@@ -13,6 +13,9 @@ src_ns: [default] src_pods: [frontend] dst_ns: [default] dst_pods: [adservice] c
 src_ns: [default] src_pods: [frontend] dst_ns: [default] dst_pods: [checkoutservice] conn: TCP 5050
 src_ns: [default] src_pods: [frontend] dst_ns: [default] dst_pods: [recommendationservice] conn: TCP 8080
 src_ns: [default] src_pods: [loadgenerator] dst_ns: [default] dst_pods: [frontend] conn: TCP 8080
+src_ns: [kube-system] src_pods: [*] dst: *.googleapis.com conn: All connections
 src_ns: [kube-system] src_pods: [*] dst: 0.0.0.0/0 conn: All connections
+src_ns: [kube-system] src_pods: [*] dst: accounts.google.com conn: All connections
+src_ns: [kube-system] src_pods: [*] dst: metadata.google.internal conn: All connections
```

Comment:
why is it changed only for `kube-system`?

Reply:
the network-policies in this test are in `synthesis_res_online_boutique.yaml` , this file specifies connections for each app (workload) in `default` namespace with a specific policy that defines the allowed egress and ingress for each with matching selectors, none of them matches and allows connections to  the new external hosts (neither to ip-blocks)
however, there are no policies defining the allowed conns from ` kube-system` so connections are allowed by default .